sdks/go/examples/wordcount/wordcount.go

```diff
@@ -93,39 +93,47 @@ var (
 	output = flag.String("output", "", "Output file (required).")
 )
 
-// Concept #3: You can make your pipeline assembly code less verbose and by
+// Concept #3: You can make your pipeline assembly code less verbose by
 // defining your DoFns statically out-of-line. A DoFn can be defined as a Go
 // function and is conventionally suffixed "Fn". The argument and return types
-// dictate the pipeline shape when used in a ParDo: for example,
+// dictate the pipeline shape when used in a ParDo. For example,
 //
-//      formatFn: string x int -> string
+//	formatFn: string x int -> string
```

Comment:
```suggestion
//	func formatFn(w string, c int) string
```

Optional: if the current notation is causing confusion, we could consider something like this (here and below)